### ISSUE TITLE
[feature suggestion] Landscape mode

### ISSUE THREAD

**Issue by ConEstArthur** [Medvedev91/timeto.me#46]
I have a lot of ideas for this and if you think this feature and the ideas are worthy enough to compliment timeto then it would an honor for me.

The app in itself at the current stage is very functional and the operations are pretty much friction less. 
I think the functionality of overall the app can be enhanced a lot by a landscape mode/ zen mode where it can be as distraction free as possible.

Now I know aesthetics is subjective but I had few suggestions for this. Not being too ambitious or anything but I really liked Nightstand mode of iphones that they launched with ios 17. (I am an android user btw)
from that you could take inspiration or you know, You do you. :D
 
![Screenshot from 2024-05-18 01-29-32](https://github.com/Medvedev91/timetome/assets/76774182/5b53d335-26c8-418f-9a8d-a3da4ea12cd6)

- Timer for **Seconds** can be eliminated from this mode as we want it to be more calm ? if that makes sense. 

Below I have attached a screenshot from an app Standby Mode Pro on Google play. It has lots of designs for some ideas.
![Screenshot_20240517-200916](https://github.com/Medvedev91/timetome/assets/76774182/405dc83d-abe4-4b26-87e7-d9acc178efbd)

- Burn-in protection for oleds would help a lot.

Or we can go "Minimalist Pomodoro Timer by Goodtime" way. 

![Screenshot from 2024-05-18 01-38-49](https://github.com/Medvedev91/timetome/assets/76774182/a66a3dfc-b178-4b9d-9f61-1c948bd1d3f6)

- fading or breathing effects can be used in the timer to make it look more serene.
- Navigation bars need to go. :) (I feel like I have asked for this umpteenth times, apologies.)

That's pretty much it. 
Thanks a lot sir. Your app makes every day easier for me. 
and It facilitates my routine tasks in its current form and will keep helping me in future updates as well.

**Comment by Medvedev91** [comment]
Initial release is here https://github.com/Medvedev91/timeto.me/releases/tag/v2026.07.31.619.

**Comment by ConEstArthur** [comment]
> Initial release is here https://github.com/Medvedev91/timeto.me/releases/tag/v2026.07.31.619.

PHENOMENAL!!!!

THANKS A LOT, SIR 🥳😭🩵

**Comment by ConEstArthur** [comment]
1. Status bar and navigation bar remains visible in zen mode if I swipe down on it to check notifications. When I swipe up to minimize the app and open it again, status bar disappears but navigation bar stays visible. Both can only be solved by closing the app from recents menu and opening it again.
2. Is there any oled burn in protection in place or planned in the future?

**Comment by Medvedev91** [comment]
> Is there any oled burn in protection in place or planned in the future?

I don't think it is an issue. Many devices support always on display, so display works full day with no problems.

**Comment by ConEstArthur** [comment]
> I don't think it is an issue. Many devices support always on display, so display works full day with no problems.

But those devices do use pixel shifting to avoid any burn ins :(

**Comment by Medvedev91** [comment]
Okay, Is this release cover this issue https://github.com/Medvedev91/timeto.me/releases/tag/v2026.09.19.624 ?

**Comment by ConEstArthur** [comment]
Yes I think we can close this issue now.

Btw in Landscape mode, on my phone when I swipe down to check status bar, it instantly disappears. So I have to race with it to check control center or notifications. 😂

**Comment by Medvedev91** [comment]
> when I swipe down to check status bar, it instantly disappears. 

Can you please make a screen's video record?

**Comment by ConEstArthur** [comment]
> Can you please make a screen's video record?

https://github.com/user-attachments/assets/52dcbb39-a0dc-4723-a0c9-2cbc9d3810af

**Comment by Medvedev91** [comment]
Hmm.. Can you please create an issue with video and device and OS information?
